lib/rules/no-unused-emit-declarations.js

```diff
@@ -332,6 +332,17 @@ module.exports = {
             emitReferenceIds
           })
         },
+        onDefineModelEnter(node, model) {
+          if (
+            node.parent &&
+            node.parent.type === 'VariableDeclarator' &&
+            node.parent.init === node
+          ) {
+            // If store the return of defineModel() in a variable, we can mark the 'update:modelName' event as used if use that variable.
+            // If that variable is unused it will already be reported by `no-unused-var` rule.
```

Comment:
```suggestion
            // If the return value of defineModel() is stored in a variable, we can mark the 'update:modelName' event as used if that that variable is used.
            // If that variable is unused, it will already be reported by `no-unused-var` rule.
```